addon/components/frost-bunsen-input-text/template.hbs

```diff
@@ -6,14 +6,25 @@
     {{/if}}
   </div>
   <div class={{inputWrapperClassName}}>
-    {{frost-text
-      class=inputClassName
-      disabled=cellConfig.disabled
-      onInput=(action "onChange")
-      placeholder=cellConfig.placeholder
-      type=inputType
-      value=renderValue
-    }}
+    {{#if (eq inputType 'password')}}
+      {{frost-password
+        class=inputClassName
+        disabled=cellConfig.disabled
+        onInput=(action "onChange")
+        placeholder=cellConfig.placeholder
+        revealable=true
```

Comment:
Do we want this to be hard-coded to `true`? Or should it be configurable via the `cellConfig`?